api/v1/model/src/commonMain/kotlin/PagedResponse.kt

```diff
@@ -33,6 +33,18 @@ data class PagedResponse<E>(
     val options: PagingOptions
 )
 
+/**
```

Comment:
Commit message:
- I guess "pages" -> "paged"
- "having to *apply* the changes"